docs/release-notes/artifacts/pr0272.yaml

```
@@ -0,0 +1,16 @@
+# --- Release notes change artifact ----
+
+schema_version: 1
+changes:
+  - title: Document release policy and schedule for haproxy monorepo
+    author: tphan025
+    type: minor
+    description: |
+      Add documentation for the release policy and schedule covering
+        both haproxy and haproxy-spoe-auth charms in the monorepo.
+    urls: 
+      pr: https://github.com/canonical/haproxy-operator/pull/272
+      related_doc: 
```

Since you updated the documentation, it's probably good to include it in the artifact. Since this artifact would be processed into a file that lives in `docs/release-notes`, I think that all you need to do is reference the landing page directly:
```suggestion
      related_doc: landing-page.md
```